lib/models/sender_service.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * A SenderService collection stores all the services that have
+ * contacted a specific user (fiscalCode) using the notification APIs
+ */
+import { pick } from "italia-ts-commons/lib/types";
+
+import * as DocumentDb from "documentdb";
+import * as t from "io-ts";
+
+import { tag } from "italia-ts-commons/lib/types";
+
+import * as DocumentDbUtils from "../utils/documentdb";
+import { DocumentDbModel } from "../utils/documentdb_model";
+
+import { FiscalCode } from "../api/definitions/FiscalCode";
+
+import { DateFromString } from "italia-ts-commons/lib/dates";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { ServiceId } from "../api/definitions/ServiceId";
+
+export const SENDER_SERVICE_COLLECTION_NAME = "sender-services";
+export const SENDER_SERVICE_MODEL_ID_FIELD = "fiscalCode";
```

Comment:
what's the difference between `ID` and `PK`?

Reply:
id is the name of the id field (ie. "id") which is used by CosmosDB as the row index.
partitionKey is needed during CosmosDB queries to let it find the partition.